synapse/push/mailer.py

```diff
@@ -131,14 +199,11 @@ def send_password_reset_mail(self, email_address, token, client_secret, sid):
             email_address (str): Email address we're sending the password
                 reset to
             token (str): Unique token generated by the server to verify
-                password reset email was received
+                the email was received
             client_secret (str): Unique token generated by the client to
                 group together multiple email sending attempts
             sid (str): The generated session ID
         """
-        if email.utils.parseaddr(email_address)[1] == "":
```

Comment:
isn't this useful? we could move it to `send_email`?

Reply:
Ah, this isn't clear from the diff at all but it's already in `send_email`:

https://github.com/matrix-org/synapse/blob/ce5f1cb98c164b5150e21d7c138f19153b4c40dd/synapse/push/mailer.py#L231-L242